e2e/jasmine/package.json

```diff
@@ -1,15 +1,12 @@
 {
   "name": "e2e-jasmine",
   "dependencies": {
-    "@bazel/jasmine": "file:../../release/npm_bazel_jasmine",
+    "@bazel/jasmine": "latest",
     "zone.js": "0.8.29"
   },
   "//": "Include an incompatible jasmine as a devDependency to verify that jasmine_node_test works regardless",
   "devDependencies": {
     "jasmine": "2.99.0",
     "jasmine-core": "2.5.2"
-  },
```

Comment:
sure. I'll add the `test` script back in so `yarn test` or `npm run test` works.

Reply:
On that note, the release script should also update the `build_bazel_rules_nodejs` http archive in the e2e tests & examples as it is no longer a local_repository but it points to the last release. This is probably better for publishing examples to their own repos. Not sure about what is best for e2e tests. `bazel test ...` from the root workspace will now test everything so what would you expect if you `cd e2e/jasmine && bazel test ...`?

npm packages @bazel/foobar are now `"latest"` since we don't want the nested repos to depend on build artifacts from the main repository (unless done automatically in bazel_integration_test). It would be inconsistent to run against `local_repository(path = "../..")` if the npm based repositories are also not based on your local mods.